### DISCUSSION TITLE
Some beginner/noob questions

### DISCUSSION THREAD

**Discussion by babsenfred** [aaronwmorris/indi-allsky#1557] [Q&A]
Hello all, new to Indi-allsky. I had TJ's Allsky up and running but decided to try Indi-Allsky. I have some questions ;

- because I had issues with finding a good picture quality in RAW daytime captures I changed to JPEG for daytime capturing. Can I leave the nighttime captures on RAW or is that a problem for creating the time-lapse videos due to the different file extensions?
- is there a ready made remote website like TJ's? Or is the remote website the same as the local website? If so, maybe I can use the TJ's remote website and upload my indi-allsky files to it? Anybody experience with that? Anybody some HOWTO tips?

Thanks a lot.

**Comment by aaronwmorris** [comment]
> Can I leave the nighttime captures on RAW or is that a problem for creating the time-lapse videos due to the different file extensions?

This is just the input file type, the output files are always JPEG (or PNG).  As long as the final image size is the same, you will be fine.

> is there a ready made remote website like TJ's?  Or is the remote website the same as the local website?

The indi-allsky local site is also the remote website.  indi-allsky supports the SyncAPI which can synchronize images and videos from your local site to a remote server.

https://github.com/aaronwmorris/indi-allsky/wiki/SyncAPI-Setup/

It is also possible to use TJ's remote website, you just have to setup the file transfers to upload to the correct folders on the remote site.  indi-allsky also generates the EndOfNight data.json file for the site.  I do not really have a howto for that, but it is possible.
